### DISCUSSION TITLE
iPhone 11 shows Passcode immediately (bypassing Face ID)

### DISCUSSION THREAD

**Discussion by richardshergold** [NiklasMerz/cordova-plugin-fingerprint-aio#308] [Q&A]
Our Ionic 5 / Capacitor application uses version 3.0.1 of this plugin.  When I test on an iPhone SE (which only has fingerprint capability) everything works ok - I see "Please Authenticate" and I am able to use the fingerprint.  However, on the iPhone 11 (which has Face ID enabled) the user is presented with the "Enter passcode" backup screen immediately i.e I do not see "Please Authenticate" at all and it does not attempt the Face ID authentication at all.  Is this a known issue with some devices?
(both devices are on iOS 14.2)

```
async isTouchAvailable() {
    let touchIDAvailable;
    try {
      touchIDAvailable = await this.faio.isAvailable();

      if (touchIDAvailable) {
        await this.presentTouch();
      }
    } catch {}
  }

  async presentTouch() {
    try {
      await this.faio.show({
        description: "Please authenticate",
      });
    } catch {}
  }
```

**Comment by NiklasMerz** [comment]
Please test the latest version

**Comment by richardshergold** [comment]
I've upgraded to 4.0.2 and I'm getting exactly the same thing on the iPhone 11 - the passcode screen comes up immediately

**Comment by NiklasMerz** [comment]
Is it just this one device? Did you try disable backup?

**Comment by richardshergold** [comment]
Hmm, setting disable backup to true fixed the issue!

**Reply by NiklasMerz** [reply]
Do you have issues with other devices as well?

**Reply by richardshergold** [reply]
Yes it was also doing the same thing on an iPhone XR but setting setting disable backup to true has fixed the issue on there as well.

**Reply by RonYaari** [reply]
I have the same issue on faio.registerBiometricSecret(), no authentication dialog popup and basically it bypass the FaceId (tested on Iphone11 & Iphone12),  faio.loadBiometricSecret() does show the FaceId dialog

**Comment by mrambow** [comment]
Had the same issue..

But totally an error on my end, forgot to set the "NSFaceIDUsageDescription" -> https://developer.apple.com/documentation/bundleresources/information_property_list/nsfaceidusagedescription
